Assets/HoloToolkit/Input/Scripts/ControllerVisualizer.cs

```diff
@@ -1,58 +1,92 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
-using System;
+using GLTF;
+using System.Collections;
 using System.Collections.Generic;
 using UnityEngine;
-using UnityEngine.VR.WSA.Input;
+#if !UNITY_EDITOR && UNITY_WSA
+using Windows.Foundation;
+using Windows.Storage.Streams;
+using Windows.UI.Input.Spatial;
+#endif
 
 namespace HoloToolkit.Unity.InputModule
 {
     /// <summary>
     /// This script spawns a specific GameObject when a controller is detected
-    /// and animates the controller position and rotation.
+    /// and animates the controller position, rotation, button presses, and
+    /// thumbstick/touchpad interactions, where applicable.
     /// </summary>
     [RequireComponent(typeof(SetGlobalListener))]
-    public class ControllerVisualizer : MonoBehaviour, ISourceStateHandler, ISourceRotationHandler, ISourcePositionHandler
+    public class ControllerVisualizer : MonoBehaviour, ISourcePositionHandler, ISourceRotationHandler
     {
         [Tooltip("Use a model with the tip in the positive Z direction and the front face in the positive Y direction.")]
         [SerializeField]
-        private GameObject controllerModel;
+        protected GameObject leftControllerOverride;
+        [Tooltip("Use a model with the tip in the positive Z direction and the front face in the positive Y direction.")]
```

Comment:
Yep, that's correct. I can clarify left/right, but I figured the GameObject names were sufficient.